scripts/nvidia-driver-versions/README.md

```diff
@@ -6,13 +6,13 @@ This script is used to generate the file and data for `[lang]/os/[version line]/
 
 This script works on Linux or MacOS and is unlikely to work in Windows. The directory structure used in the script specifically ties to bottlerocket-os/bottlerocket; it's unlikely to work in any other context.
```

Comment:
nit: it's actually tied to the bottlerocket-os/bottlerocket-core-kit repository structure.